RuleDocumentation/AvoidMultipleTypesParameter.md

```diff
@@ -0,0 +1,50 @@
+# AvoidMultipleTypesParameter
+
+**Severity Level: Warning**
+
+## Description
+
+Parameter should not have more than one type specifier.
```

Comment:
Can we also add a description why we do not want to have more than one type, i.e. what is the impact? \Just a runtime error or potentially also unpredictable or unintuitive behavior?